velox/common/base/Doubles.h

```diff
@@ -38,4 +38,12 @@ namespace facebook::velox {
 static constexpr double kMaxDoubleBelowInt64Max = 9223372036854774784.0;
 /// 2 ^ 63
 static constexpr double kMinDoubleAboveInt64Max = 9223372036854775808.0;
+
+/// For int128::max()
+///  2 ^ 127 - 2 ^ 74
+static constexpr double kMaxDoubleBelowInt128Max =
+    170141183460469212842221372237303250944.0;
+///  2 ^ 127
+static constexpr double kMinDoubleAboveInt128Max =
```

Comment:
Should we remove this as it is unused?

Reply:
We could remove it. I included it to match the int64 definitions in case it is needed later somewhere.

Reply:
I feel we can add when we need it.